### PR TITLE
change ERROR traces; while true

### PR DESCRIPTION
**Pull request by jesuspg**
#### Reviewers
@hmunfru @jframos 

#### Description
solution: bug minor 4338, and removed a 'while true' in waiting for delete

#### Testing
Locally.

### CODE REVIEW

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/ClaudiaClientOpenStackImpl.java [hunk 1]
```diff
@@ -82,34 +82,37 @@
     private TierInstanceManager tierInstanceManager = null;
     private SystemPropertiesProvider systemPropertiesProvider = null;
     private static final int POLLING_INTERVAL = 10000;
+    private static final int RETRIES = 10;        // TODO configurable parameter
```
Comment (jframos) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/ClaudiaClientOpenStackImpl.java]: I think this param should be in database (configuration_properties) and, by defult, should have a higher value.

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/ClaudiaClientOpenStackImpl.java [hunk 2]
```diff
@@ -82,34 +82,37 @@
...
+
+            boolean exists = openStackUtil.existsServerForDelete(vm.getVmid(), region, token, vdc);
+
+            if (!exists) {
+                return false;
+            }
+
+        } catch (Exception e) {
+            String errorMessage = "Error checking info from server " + vm.getVmid();
             log.error(errorMessage);
```
Comment (jframos) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/ClaudiaClientOpenStackImpl.java]: This log level should not be "error", beacuse we use this method when deleting instances and, in this case, we expect this exception.
We have to create new US to review all log levels.

**Comment by jframos** [comment]
LGTM

**Comment by hmunfru** [comment]
LGTM

**Pull request merged**